docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+
+> [!NOTE]
+> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
+
+This tutorial and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning GitHub repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes)
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+> * Understand the problem
+> * Build and Train the model
+> * Predict with the trained model
+> * Deploy and Predict with a loaded model
+
+You can find the source code for this tutorial at the [dotnet/samples](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/TransferLearningTF) repository.
+
+## Understand the problem
+
+Image classification is a common Machine Learning problem which has well-known solutions. This tutorial uses an approach that mixes new techniques (deep learning) and traditional (Logistic Regression) techniques.
+
+This model uses the [Inception model](https://storage.googleapis.com/download.tensorflow.org/models/inception5h.zip) as a *featurizer* (the model is already stored in the [assets folder](./ImageClassification.Train/assets/inputs/inception/)). This means that the model processes input images through the neural network, using the tensor output before classification. This tensor contains the *image features*, used for image identification.
+
+Finally, these image features will be fed to a Logistic regression algorithm which will learn how to classify different sets of image features.
+
+## DataSet
+
+There are two data sources: the `tsv` file and the image files.  The tags.tsv file contains two columns: the first one is defined as `ImagePath` and the second one is the `Label` corresponding to the image. The following example file does not have a header row, and looks like this:
+
+```tsv
+broccoli.jpg	broccoli
+pizza.jpg	pizza
+pizza2.jpg	pizza
+teddy2.jpg	teddy
+teddy3.jpg	teddy
+teddy4.jpg	teddy
+toaster.jpg	toaster
+toaster2.png	toaster
+```
+
+The training and testing images are located in the assets folders. These images belong to Wikimedia Commons.
+> *[Wikimedia Commons](https://commons.wikimedia.org/w/index.php?title=Main_Page&oldid=313158208), the free media repository.* Retrieved 10:48, October 17, 2018 from:  
+> https://commons.wikimedia.org/wiki/Pizza  
+> https://commons.wikimedia.org/wiki/Toaster  
+> https://commons.wikimedia.org/wiki/Teddy_bear  
+
+## ML Task - [Image Classification](https://en.wikipedia.org/wiki/Outline_of_object_recognition)
+
+To solve this problem, first you'll build an ML model. Then, train the model on existing data, evaluate the model quality, and consume the model to classify a new image.
```

Comment:
Model already exists, we are merely retraining the model with a fraction of the dataset.